src/Elliptic/Systems/ScalarGaussBonnet/Tags.hpp

```diff
@@ -0,0 +1,125 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines DataBox tags for the ScalarGaussBonnet system.
+
+#pragma once
+
+#include <string>
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/Tensor/EagerMath/DotProduct.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Elliptic/Systems/Xcts/Tags.hpp"
+#include "Evolution/Systems/CurvedScalarWave/Tags.hpp"
+#include "NumericalAlgorithms/LinearOperators/PartialDerivatives.hpp"
+#include "Options/Options.hpp"
+#include "Options/String.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Tags.hpp"
+
+/// \cond
+class DataVector;
+/// \endcond
+
+namespace sgb {
+/// Tags related to solving the scalar equation in sGB gravity.
+namespace OptionTags {
+
+struct Epsilon2 {
```

Comment:
Put these option tags into a group so they appear together in the input file. To do this, just add a `struct OptionGroup` with `static std::string name() { return "ScalarGaussBonnet"; }` and a help string. Then add `using group = OptionGroup` to the tags.